check_pgactivity

```diff
@@ -3487,6 +3490,95 @@ sub check_database_size {
 }
 
 
+=item B<extensions_versions> (9.1+)
+
+Check all extensions installed in all databases
+and raise an alarm if the current version is not the default version available
+on the instance.
+
+Typically, it is used to detect forgotten extension upgrades after package
+upgrades or a pg_upgrade.
+
+Perfdata returns the number of outdated extensions in each database.
+
+The 'postgres' database and templates are always excluded.
+
+Critical and Warning thresholds accept numbers of outdated extensions.
```

Comment:
I'm not sure that a raw number for outdated extensions has any sense.  I think we should always returns critical if there's at least one outdated extension, and provide a way to exclude an extension, and a (database, extension) from the check.

Reply:
I can imagine a case where you know that an extension will be outdated for a long time, will acknowledge a warning, and do not want to exclude the extension definitely.

So I can modify it this way:
- default 0 for critical and warning
- add --exclude parameter

Reply:
Acknowledging a warning would assume that you setup the critical threshold just above the number of expected outdated extension to get another notification if another one is outdated, and hope that an extension isn't updated while a new one is available.  Since it requires specific nagios configuration, it's less bug prone the define the explicit list of cases to ignore.